### ISSUE TITLE
Dropdown Menu Submenu Alignment

### ISSUE THREAD

**Issue by dioogbento** [zurb/foundation-sites#10881]
#### How to reproduce this bug:
1. Dropdown Menu with Submenu
2. Align-right class
3. data-alignment: 'left' option

#### What should happen:
Dropdown Menu aligned to right;
Submenu aligned to left in relation to its parent.

#### What happened instead:
Dropdown Menu aligned to right;
Submenu aligned to right in relation to its parent.

#### Browser(s) and Device(s) tested on:
Firefox, Safari and Chrome on OS X 10.10.5

#### Foundation Version(s) you are using:
6.4.3

#### Test case link:
https://codepen.io/koiastudio/pen/PEveKm

**Comment by webuxr** [comment]
I’m running to to the same issue. Posting comment as +1 and subscribing.

**Comment by Agc26** [comment]
Simbly change this css style to
```
.dropdown.menu.align-right .is-dropdown-submenu.first-sub {
    top: 100%;
    right: 0;
    left: auto;
}
```
 This ==>
```
.dropdown.menu.align-right .is-dropdown-submenu.first-sub {
    top: 100%;
    right: auto;
    left: 0;
}
```

**Comment by dgobnto** [comment]
Hi @Agc26 and thank you for your reply. I am aware you could apply such styling for a temporary fix but I posted this issue because I think it should be fixed at source.

**Comment by DanielRuf** [comment]
@SassNinja I guess this should not be there?

https://github.com/zurb/foundation-sites/blob/32378637c354145aafb8682029fca25740f3bc92/scss/components/_dropdown-menu.scss#L186-L193

**Comment by dgobnto** [comment]
@DanielRuf @SassNinja Yep, from my perspective that's not the desirable default behavior, but it's debatable (I haven't test cased extensively).

**Comment by DanielRuf** [comment]
So should I provide a PR and what should the solution look like?

**Comment by ekmoreau** [comment]
This is still not fixed?
